Adding future endeavors blog post (#530)
harvard-lil/website-static: app/_posts/2024-03-29-new-endeavors-at-the-library-innovation-lab.md

---
title: New Endeavors at the Library Innovation Lab

> author: aadi-kulkarni

Reviewer: Is this the right author?

---

> This month, the Library Innovation Lab celebrated the [full and unqualified release](https://lil.law.harvard.edu/blog/2024/03/26/transitions-for-the-caselaw-access-project/) of the Caselaw Access Project data. We took the opportunity to gather and look to the future at [Transform: Justice on March 8th](http://lil.law.harvard.edu/about/cap-celebration). During that event we were reminded of what we already knew: the open legal data movement is alive and well. 

Reviewer: ```suggestion
This month, the Library Innovation Lab celebrated the [full and unqualified release](https://lil.law.harvard.edu/blog/2024/03/26/transitions-for-the-caselaw-access-project/) of the Caselaw Access Project data. We took the opportunity to gather and look to the future at [Transform: Justice on March 8th](http://lil.law.harvard.edu/about/cap-celebration). The event reminded us of what we already knew: the open legal data movement is alive and well. 
```
Maybe change from passive to active in the last sentence, though I'm not sure I like it much better.

> In addition to hearing the story of CAP, there were also conversations between librarians, vendors, and advocates about future paths for accessing the law. Noted open-access activist Carl Malamud made his pitch for a [Full Faith and Credit Act](https://www.youtube.com/watch?v=JTIIfCp1Ufk). 

Reviewer: ```suggestion
In addition to hearing the story of CAP, librarians, vendors, and advocates had conversations about future paths for accessing the law. Noted open-access activist Carl Malamud made his pitch for a [Full Faith and Credit Act](https://www.youtube.com/watch?v=JTIIfCp1Ufk). 
```

> As we see CAP move on to a new chapter, LIL is excited to bring the same experimental mindset that allowed for such an ambitious project into other endeavors. 

Reviewer: Being "excited" is overused in general, and in this post -- it's the first of two uses in three paragraphs. I think it would be safe to err on the side of never using this phrase.

Author: 😂 fair enough, and we're in special danger of it with posts like this where I'm pulling copy from a couple of other places.

Here’s some of what’s coming up next at the Library Innovation Lab: 

### Facilitating open, equitable access to knowledge

> **[COLD Cases](https://huggingface.co/datasets/harvard-lil/cold-cases)**: We’re excited to be collaborating with the [Free Law Project](http://www.free.law) on a couple of projects. First, we worked with FLP to create Cold Cases, a pipeline for standardizing and sharing bulk legal data. And now that it is able to fully leave the nest, to consolidate efforts and continue to provide access to CAP’s data it will now live alongside FLP’s other extensive resources. 

Reviewer: ```suggestion
**[COLD Cases](https://huggingface.co/datasets/harvard-lil/cold-cases)**: We’re excited to be collaborating with the [Free Law Project](http://www.free.law) on a couple of projects. First, we worked with FLP to create COLD Cases, a pipeline for standardizing and sharing bulk legal data. And now that it is able to fully leave the nest, to consolidate efforts and continue to provide access to CAP’s data it will now live alongside FLP’s other extensive resources. 
```
The last sentence is unclear, and feels like it's burying the lede.

> **[Teraflop & CAP](https://threadreaderapp.com/thread/1766157444936507837.html)**: It’s great to already see work by the AI community in processing the CAP dataset for legal understanding, such as this work by Teraflop to generate text embeddings and search indexes for the data.

Reviewer: ```suggestion
**[Teraflop & CAP](https://threadreaderapp.com/thread/1766157444936507837.html)**: It’s great to see work by the AI community in processing the CAP dataset for legal understanding, such as this work by Teraflop to generate text embeddings and search indexes for the data.
```
Even though splitting an infinitive isn't strictly forbidden, it's awkward here.

> **Future Collaborations**: We see huge potential for Harvard to be a neutral platform that can help release data that helps the whole industry. We are having interesting conversations with companies like Westlaw, LexisNexis, and vLex that see potential to this kind of partnership, and we hope to make more connections like that soon.

Reviewer: ```suggestion
**Future Collaborations**: We see potential for Harvard to be a neutral platform for releasing data that helps the whole industry. We are having conversations with companies like Westlaw, LexisNexis, and vLex that see potential in this kind of partnership, and we hope to make more connections soon.
```

 
### Empower everyone – AI, understanding, access to justice

> **[Library Innovation Lab Artificial Intelligence Fund](http://lil.law.harvard.edu/about/ai)**: Lexis Nexis is the first funder of our AI fund, which has been set up to support exploration at the edge of law, AI technology, libraries, and society.

Reviewer: ```suggestion
**[Library Innovation Lab Artificial Intelligence Fund](http://lil.law.harvard.edu/about/ai)**: LexisNexis is the first funder of our AI fund, which has been set up to support exploration at the edge of law, AI technology, libraries, and society.
```

> **[Open Legal AI Workbench](https://lil.law.harvard.edu/blog/2024/03/08/announcing-the-open-legal-ai-workbench-olaw/)**: There will be lots of innovation and creation in the legal AI space in the near future. Our recently released Open Legal AI Workbench is a simple, well-documented, and extensible framework for legal AI researchers to build services using tool-based retrieval augmented generation. 

Reviewer: ```suggestion
**[Open Legal AI Workbench](https://lil.law.harvard.edu/blog/2024/03/08/announcing-the-open-legal-ai-workbench-olaw/)**: Our recently released Open Legal AI Workbench is a simple, well-documented, and extensible framework for legal AI researchers to build services using tool-based retrieval augmented generation. 
```

> ### Leveling up both of these approaches

Reviewer: I might lose this heading, and just make the next item a paragraph.

> **Institutional Data Initiative**: Keep your eyes peeled for more news about an endeavor we have in the works to create a trusted conduit for helping data get from where it is to where it needs to be. The Institutional Data Initiative aims to scale the types of collaborations we discussed on Friday between knowledge institutions and industry partners. High-quality corpora and collection expertise paired with industry resources stands to grow knowledge and the public interest in lockstep with AI.

Reviewer: ```suggestion
Keep your eyes peeled for more news about the Institutional Data Initiative, a plan to create a trusted conduit for legal data for AI. By pairing high-quality corpora and collection expertise with industry resources, it aims to scale the collaborations between knowledge institutions and industry partners that we discussed at Transform: Justice. The goal is to grow both knowledge and the public interest.
```
Maybe?
